scripts/export_vhi.py

```diff
@@ -0,0 +1,33 @@
+from pathlib import Path
+
+import sys
+sys.path.append('..')
+from src.exporters import CDSExporter  # noqa
+
+
+def export_vhi_2018():
```

Comment:
@tommylees112 , does this look right? I get the following:

```
2019-05-12 15:30:56,184 INFO Sending request to https://cds.climate.copernicus.eu/api/v2/resources/reanalysis-era5-single-levels
/Users/gabrieltseng/anaconda3/envs/esowc-drought/lib/python3.7/site-packages/urllib3/connectionpool.py:847: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
2019-05-12 15:30:56,591 INFO Request is queued
/Users/gabrieltseng/anaconda3/envs/esowc-drought/lib/python3.7/site-packages/urllib3/connectionpool.py:847: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
/Users/gabrieltseng/anaconda3/envs/esowc-drought/lib/python3.7/site-packages/urllib3/connectionpool.py:847: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
```
Followed by loads of insecure request warnings. I can't tell if this is expected behaviour, and if I just need to wait longer for the request to kick off.

Reply:
I get loads of these too - there is a way to fix:

1) Add this into your imports
```
import certifi
import urllib3
http = urllib3.PoolManager(
    cert_reqs='CERT_REQUIRED',
    ca_certs=certifi.where()
)
```

2) Change the `.cdsapirc` file
- changed the verify property from 0 to 1 

Based on answers from [here](https://github.com/ecmwf/cdsapi/issues/4)

NOTE: his downloads is still not working so maybe this doesn't help...

Reply:
Adding the error for reference:

```
mars - INFO   - 20190512.194213 - Requesting any number of fields (request describes 52560)
mars - INFO   - 20190512.194213 - Calling mars on 'marscore', callback on 58915
mars - INFO   - 20190512.194213 - Callback at address 10.10.10.144, port 58915
mars - INFO   - 20190512.194215 - Mars client is on compute-0003.cdsprod.copernicus-climate.eu (10.10.10.144) 58915
mars - INFO   - 20190512.194215 - Mars server is on marscore (10.10.10.8) 36571
mars - INFO   - 20190512.194215 - Server task is 893 [mars]
mars - INFO   - 20190512.194215 - Request cost: 514,650 fields, 462.346 Gbytes online, nodes: mvr0000 mvr0001 mvr0002 mvr0003 mvr0004 mvr0005 mvr0006 mvr0007 mvr0008 [mars]
mars - ERROR  - 20190512.194243 - Mars server task finished in error
mars - ERROR  - 20190512.194243 - Assertion failed: handle_[n] != 0 in dataHandle, line 94 of /home/cds/git/mars-server/mars-server/src/dhslib/DSRetrieve.cc [mars]
mars - ERROR  - 20190512.194243 - Error code is -2
mars - ERROR  - 20190512.194243 - Request failed
mars - INFO   - 20190512.194243 - Request time:  wall: 31 sec
mars - INFO   - 20190512.194243 -   Processing in mars: wall: 30 sec
mars - INFO   - 20190512.194243 -   Visiting mars: wall: 30 sec
mars - INFO   - 20190512.194243 - Memory used: 103.04 Mbyte(s)
mars - ERROR  - 20190512.194243 - Some errors reported
```

Reply:
Fixed!